src/Catalog/Persistence/Storage.cs

```diff
@@ -170,6 +170,17 @@ public Uri ResolveUri(string relativeUri)
             return new Uri(BaseAddress, relativeUri);
         }
 
+        /// <summary>
+        /// It will return true if there are changes between the source and destination.
```

Comment:
Isn't it the opposite? The implementation above returns true if no changes and false if changes.

Reply:
Correct - updated.